src/redux/saga/getDataTodoItemsSaga.ts

```diff
@@ -0,0 +1,28 @@
+import { FirebaseDatabaseTypes } from '@react-native-firebase/database';
+import { createReferenceHelper } from '../../helpers/createReferenceHelper';
+import { TodoItemType } from '../../models';
+import { SignInPayload } from '../actions/authActions/successSignInAction';
+import { put, select } from 'redux-saga/effects';
+import { userTokenSelector } from '../selectors/userTokenSelector';
+import { dataTodoItemsAction } from '../actions/todoActions/dataTodoItemsAction';
+import { GetDataTodoItemsSagaAction } from '../actions/todoSagaActions/getDataTodoItemsSagaAction';
+
+export function* getDataTodoItemsSaga(action: GetDataTodoItemsSagaAction) {
+  try {
+    const { setLoading } = action.payload;
```

Comment:
your code different in every saga 

in _deleteItemSaga_  you use selector and getting data from payload above try/catch 

here in try/catch 

easier understand your code if you will write in one style

Reply:
understand